src/main/java/edu/harvard/dbmi/avillach/dictionary/dashboarddrawer/DashboardDrawerController.java

```diff
@@ -0,0 +1,24 @@
+package edu.harvard.dbmi.avillach.dictionary.dashboarddrawer;
```

Comment:
What's the difference between `edu.harvard.dbmi.avillach.dictionary.dashboarddrawer` and `edu.harvard.dbmi.avillach.dictionary.dashboard` as far as business purpose goes? Are these legitimately different domains, or is this a BDC specific implementation of the same domain?

Reply:
Yea I was in the same boat at the beginning and was trying to fit it into to the Dashboard thus why it was basically a replication of it until changed the class names. 

I think I should follow up on this ticket with Cleaning up some of the dashboard method and looking at how this integrates with the dashboard more closely.  

From more of a backend mindset I think it should relate to a dataset endpoint and not a feature endpoint.